src/steps/active-directory/__tests__/converters.test.ts

```diff
@@ -20,21 +20,35 @@ import exampleOrganization from './fixtures/exampleOrganization';
 
 describe('createAccountEntityWithOrganization', () => {
   test('properties transferred', () => {
+    const exampleExtraData = {
+      mobileDeviceManagementAuthority: 'mobileDeviceManagementAuthority',
+      subscriptionState: 'subscriptionState',
+      intuneAccountID: 'intuneAccountID',
+    };
     const accountEntity = createAccountEntityWithOrganization(
       exampleIntegrationInstance,
       exampleOrganization,
+      exampleExtraData,
     );
     expect(accountEntity).toEqual({
       _class: ['Account'],
       _key: 'microsoft_365_account-the-instance-id',
       _type: 'microsoft_365_account',
-      _rawData: [{ name: 'default', rawData: exampleOrganization }],
+      _rawData: [
+        {
+          name: 'default',
+          rawData: { ...exampleOrganization, ...exampleExtraData },
+        },
+      ],
```

Comment:
I think it worthwhile to consider storing the raw data like this:

```suggestion
      _rawData: [
        {
          name: 'default',
          rawData: exampleOrganization,
        },
        {
          name: 'intune',
          rawData: exampleIntuneConfig,
      ],
```